drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -826,14 +805,46 @@ function DragAndDropBlock(runtime, element, configuration) {
         return feedback_msgs_list.map(function(message) { return message.message; }).join('\n');
     };
 
-    var updateDOM = function(state) {
+    var updateDOM = function() {
         var new_vdom = render(state);
         var patches = virtualDom.diff(__vdom, new_vdom);
         root = virtualDom.patch(root, patches);
         $root = $(root);
         __vdom = new_vdom;
     };
 
+    // Uses edX JS accessibility tools to read feedback messages when present.
+    var readScreenReaderMessages = function() {
+        if (window.SR && window.SR.readText) {
```

Comment:
A thought about this-- I think you mentioned that the drag and drop block is used outside of the LMS. In those environments, you wouldn't have any aria-live text for feedback, which would be an accessibility problem.

You might wish to consider adding your own "screen reader" region with the HTML of the drag and drop problem and send your feedback messages there instead of to the edX one.